src/pamiq_core/threads/thread_types.py

```diff
@@ -0,0 +1,18 @@
+from enum import Enum, auto
+
+
+class ThreadTypes(Enum):
+    CONTROL = auto()
+    INFERENCE = auto()
+    TRAINING = auto()
+
+    @property
+    def name(self) -> str:
+        return THREAD_NAMES[self]
+
+
+THREAD_NAMES = {
```

Comment:
この定数は`thread_types`モジュールの中からのみ使用される想定のため、`_`をprefixにつけてprivate扱いにしましょう。（本当にprivateになるわけではないですが、pythonの規約的にそうなっています）
```suggestion
_THREAD_NAMES = {
```

`_THREAD_NAMES`オブジェクト自体は編集可能で、値を置き換えられてしまう可能性があるので、このようにしておく必要があります。

Reply:
なるほど！　確かに！
命名まで変えるのは良くなかったですね。

ありがとうございます！　修正します！